plugins/ssl/certificate_file_expiry

```diff
@@ -72,23 +87,70 @@ if [ "$1" = "config" ] ; then
   echo "graph_category security"
 fi
 
+if [ "${warning:-}" = "" ] ; then
+  # as default when certificate is only valid for 5 days or less emit a warning
+  warning="5:"
+fi
+if [ "${critical:-}" = "" ] ; then
+  # as default when certificate is only valid for 1 day or less emit a critical
+  critical="1:"
+fi
+
 now=$(date +%s)
+get_validity() {
+  openssl_type=$1
```

Comment:
I would suggest to use `local` here.
(please note, that you need to run `shellcheck` with `-s dash` as a parameter in order to receive proper hints)

Reply:
thanks for the hint, yes my shellcheck is still configured for posix as my IDE currently doesn't support configuring dash for sh shebang: https://github.com/BashSupport-Pro/bashsupport-pro/issues/91